src/applications/static-pages/facilities/vet-center/components/VAFacilityPhone.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const nonNumericReplacement = new RegExp('[^0-9\\+\\-]+', 'g');
+export const processPhoneNumber = phoneNumber => {
```

Comment:
**Praise:** this is a smart use of regex to more reliably separate the phone number from the extension 👏